src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -38,7 +38,7 @@ import {
   createVersion,
   editMapping,
 } from '../../../redux/actions/dictionaries/dictionaryActionCreators';
-import dictionaries from '../../__mocks__/dictionaries';
+import dictionaries, { threeMockDictionaries } from '../../__mocks__/dictionaries';
```

Comment:
The name does not have to include the number of items contained

Reply:
Understood. Changing it now.

Reply:
I've done so